common/src/main/java/com/wynntils/utils/wynn/InventoryUtils.java

```diff
@@ -20,6 +24,13 @@ public final class InventoryUtils {
     public static final int SOUL_POINTS_SLOT_NUM = 8;
     public static final int INGREDIENT_POUCH_SLOT_NUM = 13;
 
+    private static final int RING_1_SLOT_NUM = 9;
+    private static final int RING_2_SLOT_NUM = 10;
+    private static final int BRACELET_SLOT_NUM = 11;
+    private static final int NECKLACE_SLOT_NUM = 12;
+    private static final int[] ACCESSORY_SLOTS = {RING_1_SLOT_NUM, RING_2_SLOT_NUM, BRACELET_SLOT_NUM, NECKLACE_SLOT_NUM
+    };
```

Comment:
Unfortunate formatting

Reply:
Yes, I agree, but it is upheld by Spotless. :-( I tried fixing it but Spotless restored it to this look again. I need to make some code changes to force Spotless to change the formatting -- is it worth it?